diff -rubB ixgbevf-3.1.2-orig/src/kcompat.h ixgbevf-3.1.2/src/kcompat.h
--- ixgbevf-3.1.2-orig/src/kcompat.h	2016-05-31 19:57:07.216072787 +0000
+++ ixgbevf-3.1.2/src/kcompat.h	2016-05-31 19:58:07.604251570 +0000
@@ -3396,7 +3396,8 @@
 }
 #define vzalloc(_size) _kc_vzalloc(_size)

-#ifndef vlan_get_protocol
+#if (!(RHEL_RELEASE_CODE >= RHEL_RELEASE_VERSION(5,7)) || \
+     (RHEL_RELEASE_CODE == RHEL_RELEASE_VERSION(6,0)))
 static inline __be16 __kc_vlan_get_protocol(const struct sk_buff *skb)
 {
 	if (vlan_tx_tag_present(skb) ||
